draw2dimg/ftgc.go

```diff
@@ -58,6 +58,8 @@ func NewGraphicContext(img draw.Image) *GraphicContext {
 	switch selectImage := img.(type) {
 	case *image.RGBA:
 		painter = raster.NewRGBAPainter(selectImage)
+	case *image.Gray:
+		painter = raster.NewGRAYPainter(selectImage)
```

Comment:
GRAY should be Gray. The RGBA above is not Rgba as per https://github.com/golang/go/wiki/CodeReviewComments#initialisms but Gray is not an initialism.